Example/Controllers/SwiftUI/Features/Single/MiniAppSingleView.swift

```diff
@@ -168,6 +170,18 @@ struct MiniAppSingleView: View {
                 sendResumeMiniApp(miniAppId: viewModel.miniAppId, version: viewModel.miniAppVersion)
             }
         })
+        .onChange(of: viewModel.shouldCloseMiniApp.value, perform: { newValue in
+            if newValue {
+                if viewModel.closeWithConfirmation {
+                    closeAlertMessage = MiniAppAlertMessage(
+                        title: "Close Miniapp",
+                        message: "Are you sure, you want to close this MiniApp?"
+                    )
```

Comment:
You should not use the hardcoded close confirmation alert message here, rather you should the one which is set the mini-app itself using setCloseAlert